Big cleanup update

valnub opened the pull request:
- Deleted blogs that haven't been updated for 6 months
- Updated URLs of blogs that have moved to a new location
- Added new fancy frontend-related blogs (including mine ;-)), mostly techy but also added a little flavor of UI design since I believe that devs should care about how their UIs look and feel.

I hope I didn't accidentally f*ck up anything. The file works well for me on Feedly.

**Ps:** Note that the diff might look like as if I removed all the old feeds. That isn't the case. Feedly does not sort the feeds alphabetically so the order of the feeds in the export file is a whole lot different compared to your original one. Maybe you have an old instance of Google Reader somewhere so you can import, sort and then re-export the file? ;-)

Commit:
Big cleanup update

- Deleted blogs that haven't been updated for 6 months
- Updated URLs of blogs that have moved
- Added new fancy frontend-related blogs

Commit:
Removed Reddit Feeds

Not much activity there, so I removed them and replaced with actual
blogs

paulirish commented:
wooo! awesome. thank you.

can you do me a favor?

take the XML before your PR, and sort the entries. store as A.xml
take the XML after your PR, and sort the entires. store as B.xml

put up a gist of the A/B diff

that'd definitely help me and others reviewing this to see which are removed vs moved.

valnub commented:
Sure thing, here you go: https://gist.github.com/valnub/c91c9f695009b4ac5813

Commit:
Removed duplicate Addy Osmani blog

Commit:
Removed broken + unrelated feeds

- Removed Tantarek
- Removed Photoshop Lady

Commit:
Minor fixes

- Removed 4 duplicates
- Added 1 new feed "JavaScript scene"

Commit:
Changed sorting, added new feeds

- Changed sorting to it's identical to the gist:
https://gist.github.com/valnub/c91c9f695009b4ac5813
- Added 4 new js feeds

Commit:
Removed Pinboard

Commit:
Updated modified date value

paulirish commented:
I cleaned this up a bit so its easier to read: https://gist.github.com/anonymous/01af86f1661d0b26fb63

paulirish commented:
theres a few removals in here i dont agree with:
- alexsexton
- andismith
- antimatter
- badassjs
- blog.mozilla.org/bjacob
-     blog.mozilla.org/javascript/
- blog.patrickmeenan.com/
-      blog.tojicode.com/
- bricss.net
- calendar.perfplanet.com
- creativejs.com
- eligrey.com/blog
- erik.eae.net
- mikewest.org
- peter.sh
- paulrouget.com
- stephanierieger.com
- tomdale.net
- tjholowaychuk.com

paulirish commented:
there are also some additions I don't care for, but that's a bit more subjective.

Can you restore the above removals?

ZackBoe commented:
A couple I'd like to see stick around:
24ways.org
garann.com
On Oct 20, 2015 3:32 PM, "Paul Irish" notifications@github.com wrote:

> there are also some additions I don't care for, but that's a bit more
> subjective.
> 
> Can you restore the above removals?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/paulirish/frontend-feeds/pull/4#issuecomment-149676821
> .

Commit:
Re-added previously removed feeds

valnub commented:
@paulirish Sure, I have re-added the feeds you listed, but are you really, **_really**_ sure that you'd like to keep em? Some are gone (404) and some haven't been updated for a looooong time (more than 1 year). I feel like a zombie-blog necromancer now :)

```
Last post  |  URL
-----------------------------------------------------------------
404        |  https://blog.mozilla.org/bjacob
404        |  http://stephanierieger.com/feed/ (Website is still there but feed is gone?)
May 2013   |  http://eligrey.com/blog
Sep 2013   |  https://mikewest.org
Dec 2013   |  http://peter.sh
Jan 2014   |  http://badassjs.com
May 2014   |  http://www.andismith.com
May 2014   |  http://antimatter15.com/wp
Jul 2014   |  http://blog.tojicode.com
Sep 2014   |  http://blog.patrickmeenan.com
Sep 2014   |  http://erik.eae.net
Oct 2014   |  http://creativejs.com
```

Also, please note that some sites still exist but their **_feeds**_ are either gone or moved. I tried to get the updated URL where ever possible but some sites simply stopped supporting them. I also noticed that some blogs actually have recent updates but their feeds are not maintained any more (but not removed) and thus out-of-date. There's nothing I could do about those. They're dead, Jim.

valnub commented:
@ZackBoe Those 2 sites have their last posts from end of 2014. Are those still alive?

ZackBoe commented:
I'm not sure about Garann's blog, but 24ways posts each December. It might
seem a bit weird, but their content is quite good. Up to Paul though.

On Tue, Oct 20, 2015 at 7:24 PM, Timo Ernst notifications@github.com
wrote:

> @ZackBoe https://github.com/ZackBoe Those 2 sites have their last posts
> from end of 2014. Are those still maintained?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/paulirish/frontend-feeds/pull/4#issuecomment-149731551
> .

Commit:
Removed 1 non-tech blog

Commit:
Added rreserver js blog

paulirish commented:
merged. :)  thanks for the followups
sorry for the delay bud!

valnub commented:
Nice :+1: 
No problem, Paul :)
